proto/datastore.d.ts

```diff
@@ -1,5 +1,5 @@
 import * as $protobuf from "protobufjs";
-import * as long from "long";
+import * as Long from "long";
```

Comment:
Yarrr, nice find and sorry for the mixup.  What you *really* wanna do here is update this line in package.json:

```
"proto:datastore": "mkdir -p proto && pbjs -t static-module -w commonjs -p node_modules/google-proto-files google/datastore/v1/datastore.proto | pbts -i long -o proto/datastore.d.ts -",
```

Make that:
```
"proto:datastore": "mkdir -p proto && pbjs -t static-module -w commonjs -p node_modules/google-proto-files google/datastore/v1/datastore.proto | pbts -i Long -o proto/datastore.d.ts -",
```

That way the fix doesn't get roughed up by the generator.

Reply:
Then it's `import Long from 'Long'` which I guess will go boom on linux?

Reply:
Nah, it'll probably be fine

Reply:
This very much will blow up on Linux -- import filenames are case sensitive there.  Applying the suggestion results in:
```
> @google-cloud/datastore@3.1.2 compile /home/mgl/3psrc/nodejs-datastore
> tsc -p . && cp -r src/v1 build/src && cp -r proto* build && cp test/*.js build/test

proto/datastore.d.ts:2:23 - error TS2307: Cannot find module 'Long'.

2 import * as Long from "Long";
                        ~~~~~~


Found 2 errors.
```

Reply:
Is this what we want? https://github.com/dcodeIO/protobuf.js/pull/1166

Reply:
Yes lol.  That would fix all the things.